src/main/plugins/org.dita.base/plugin.xml

```diff
@@ -91,6 +91,7 @@ See the accompanying LICENSE file for applicable license.
       <val>NUMTITLE</val>
     </param>
     <param name="args.filter" desc="Specifies a filter file to be used to include, exclude, or flag content." type="file"/>
+    <param name="args.filter.list" desc="Specifies a list of filter files to be used to include, exclude, or flag content." type="file"/>
```

Comment:
Still need to remove this, left over from original commit that started the pull request.